### PR TITLE
Feature 3.6/parametrize truncate collect

### PR DESCRIPTION
**Pull request by dothebart**
backport of https://github.com/arangodb/arangodb/pull/12857

### CODE REVIEW

**Comment by dothebart** [comment]
http://jenkins.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/12337/

**Comment by dothebart** [comment]
http://jenkins.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/12339/

**Comment by dothebart** [comment]
tests blue except unrelated arangosh -testing.js garbage collection crash. 
re-running mac because of git connectivity issues.
http://jenkins.arangodb.biz:8080/job/arangodb-matrix-pr-mac-onlycatch/9773/

**Comment by dothebart** [comment]
mac tests blue.

**Review by jsteemann — APPROVED** [review]

File: js/client/modules/@arangodb/arango-collection.js [hunk 1]
```diff
@@ -568,6 +570,8 @@ ArangoCollection.prototype.truncate = function (options) {
       // ignore any WAL-flush related error (may be a privilege issue anyway)
     }
   }
+  return requestResult;
+
```
Comment (jsteemann) [inline js/client/modules/@arangodb/arango-collection.js:574]:
```suggestion
```

**Comment by KVS85** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/12533/

**Comment by KVS85** [comment]
Tests blue.

**Pull request merged**